### ISSUE TITLE
README link to overview of BESSPIN Voting System broken

### ISSUE THREAD

**Issue by stanleybak** [GaloisInc/BESSPIN-Voting-System-Demonstrator-2019#2]
The top-level README has several broken links. For example the text "All testers should begin by reading the Overview of BESSPIN Voting System" links to https://github.com/GaloisInc/BESSPIN-Voting-System-Demonstrator-2019/blob/master/Overview-of-BESSPIN-Voting-System.pdf which does not exist.

I did find the document here (notice it's now in the docs folder): https://github.com/GaloisInc/BESSPIN-Voting-System-Demonstrator-2019/blob/master/docs/Overview-of-BESSPIN-Voting-System.pdf

Similarly, the readme text "The top-level directory Attacker-Quickstart" has a broken link to the quickstart (I also don't see it immediately in the docs folder).

It would be helpful if the links were fixed.

**Comment by kiniry** [comment]
Thanks for poking around and noticing this, Stan!  I'll see what we can
do.  As with all artifacts snapshotted in the past, sometimes this happens.

The evolution of this particular project and repo, which was funded for a
few more months past the DEF CON deliverable, is not public but is hosted
on Galois's external facing GitLab.  Let me know if you'd like access and
I'll see what I can do.  Perhaps I should just push a version of it to Free
& Fair's GitHub Organization.

Joe

**Comment by stanleybak** [comment]
Thanks Joe. We're trying to get some students to get their heads around the verification that was done for this project (and maybe some other ones you provided as well). 

Do you have some advice on the best way to understand the process that was used to create / verify this system? For example, I see various verification artifacts but it's not immediately clear to me how they're connected. I suppose once things are formalized the `Makefile` is the best reference to see how things connect? Is this what the assurance case is meant to provide? I was thinking a picture like this, but more complete:

![connections](https://user-images.githubusercontent.com/10566441/218560527-c0f8c37e-1310-46b5-b9ae-bd5f35c57f1a.png)

**Comment by kiniry** [comment]
You are right, we never did a coherent write-up of the joint assurance case of the project, given how quickly we had to do the work to demonstrate it at DEF CON.  The easiest thing to do is to book 60-90 with me and I can walk you and the students through it.
